sdk/include/platform/arty-a7/platform-entropy.hh

```diff
@@ -9,6 +9,7 @@ DECLARE_AND_DEFINE_INTERRUPT_CAPABILITY(RevokerInterruptEntropy,
                                         InterruptName::EthernetReceiveInterrupt,
                                         true,
                                         false)
+#define REVOKER_INTERRUPT_ENTROPY STATIC_SEALED_VALUE(RevokerInterruptEntropy)
```

Comment:
Why a define for a thing that is used only once?

Reply:
I found it more readable like that. I can remove.